docs/install_guides/installing-openvino-yum.md

```diff
@@ -24,53 +24,63 @@ Installing OpenVINO Runtime from YUM is recommended for C++ developers. If you a
 
 .. tab:: Processor Notes
 
-  Processor graphics are not included in all processors.
-  See `Product Specifications`_ for information about your processor.
+   Processor graphics are not included in all processors.
+   See `Product Specifications`_ for information about your processor.
 
-  .. _Product Specifications: https://ark.intel.com/
+   .. _Product Specifications: https://ark.intel.com/
 
 .. tab:: Software
 
-  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
-  * GCC 8.2.0
-  * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/>`_
+   * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+   * GCC 8.2.0
+   * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/>`_
 
 @endsphinxdirective
 
 ## Install OpenVINO Runtime
 
 ### Step 1: Set Up the Repository
 
-1. Create the YUM repo file in the `/tmp` directory as a normal user:
-   ```
-   tee > /tmp/openvino-2022.repo << EOF
-   [OpenVINO]
-   name=Intel(R) Distribution of OpenVINO 2022
-   baseurl=https://yum.repos.intel.com/openvino/2022
-   enabled=1
-   gpgcheck=1
-   repo_gpgcheck=1
-   gpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
-   EOF
-   ```
-2.	Move the new openvino-2022.repo file to the YUM configuration directory `/etc/yum.repos.d`:
-   ```sh
-   sudo mv /tmp/openvino-2022.repo /etc/yum.repos.d
-   ```
-3.	Verify that the new repo is properly setup by running the following command:
-   ```sh
-   yum repolist | grep -i openvino
-   ```
-    You will see the available list of packages.
+@sphinxdirective
 
+1. Create a YUM repository file (`openvino-2022.repo`) in the `/tmp` directory as a normal user:
 
-To list available OpenVINO packages, use the following command:
+   .. code-block:: sh
 
-@sphinxdirective
+      tee > /tmp/openvino-2022.repo << EOF
+      [OpenVINO]
+      name=Intel(R) Distribution of OpenVINO 2022
+      baseurl=https://yum.repos.intel.com/openvino/2022
+      enabled=1
+      gpgcheck=1
+      repo_gpgcheck=1
+      gpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+      EOF
+
+2. Move the new `openvino-2022.repo` file to the YUM configuration directory, i.e. `/etc/yum.repos.d`:
+   
+   .. code-block:: sh
+
+      sudo mv /tmp/openvino-2022.repo /etc/yum.repos.d
+
+3. Verify that the new repository is set up properly.
 
    .. code-block:: sh
 
-      yum list 'openvino*'
+      yum repolist | grep -i openvino
+
+   You will see the available list of packages.
+
+   .. note::
+
+      If you encounter an error while running this command on CentOS, check the :ref:`Troubleshooting Guide <yum troubleshooting 1>` for the possible solution.
```

Comment:
our RPM packages do not work on CentOS since they are built on RHEL 8.2
We support RHEL 8.2+ and other compatible systems like

Reply:
Updated. Please check. :)